### PR TITLE
Release/wandb 0.11.0

### PR DESCRIPTION
**Pull request by dannygoldstein**
n/t

### CODE REVIEW

**Review by KyleGoyette — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+#### :bug: Bug Fix
+
+- Fix issue where `wandb.watch()` broke model saving in pytorch
+- Fix issue where uniform sweep parameters were parsed as int_uniform
+- Fix issue where file_stream thread was killed on 4xx errors
+
+#### :nail_care: Enhancement
+
+- Improve performance of artifact logging by making it non-blocking
+- Add wandb integration for Stable-benchmarks-3
```
Comment (araffin) [inline CHANGELOG.md:16]: should be `Stable-Baselines3`, no?

Comment (raubitsj) [reply CHANGELOG.md:16]: oops apologies @araffin
Fixed in https://github.com/wandb/client/pull/2402

Comment (dannygoldstein) [reply CHANGELOG.md:16]: oops !

**Pull request merged**